mission/mission_core/missions/ScanTheCode_mission.py

```diff
@@ -0,0 +1,249 @@
+'''
+A simple mission template for creating new missions.
+
+The variables and functions defined here must exist in all mission classes.
+Comments are provided to explain the purpose of each variable and function and can be removed.
+'''
+
+from typing import Dict, Tuple
+from mission_core import PositionData
+from perception_core import CameraData, Results
+from itertools import groupby
+
+
+class SimpleMission:
+
+    # Define the initial perception commands here (if any)
+    # This dictionary must exist, but some commands are entered as examples
+    init_perc_cmd = {
+        "start": ["center"],
+        "stop": ["port", "starboard"],
+        "record": ["center"],
+        "stop_record": ["port", "starboard"],
+        "load_model": [("center", "ML_Resources/stcA.pt")],
+        "stop_model": ["port", "starboard"]
```

Comment:
When you "stop" a camera (the second arg) it will also stop the recording and model if either is running. So you don't need to pass it again for the "stop_record" and "stop_model".